### DISCUSSION TITLE
Useless in combination with Ngnix-Proxy or anything else that uses port 80

### DISCUSSION THREAD

**Discussion by AriMcBrown** [nextcloud/all-in-one#1854] [Ideas] [answered]
I was so hyped about the new AIO release, to finally have a opportunity to play with the next generation of NextCloud.

I was making preparations to backup my old instance and prepare for the new AIO solution.

But I couldn't roll it out through Portainer (stacks), I couldn't use directories in stead of volumes, it seems to be very greedy trying to claim port 80, 8080 and 8433 (the last 2 are fine imo)

And by working my way through deeper parts of the documentation the sad realization is dawning on me that it will never run om my server. Because of how it needs all these specific things to work properly. I'm very ... very deeply saddened and surprised by the sheer arrogance 1 single (at least before deployment) needs to claim resources that are so common practice in using with docker.

The ability to deploy on an existing server is near impossible

**Comment by Zoey2936** [comment, marked as answer]
For the ports, did you read the reverse proxa docs: https://github.com/nextcloud/all-in-one/blob/main/reverse-proxy.md ?
And for the volumes, yes you are forced  to use docker volumes and I see there no problem in using them,  but you can use a custom nextcloud data directory, if you realy need bindings instead of volumes, you could try to create the volumes manually using "docker volume create" and create them as binding, not volume, that should be written somewhere in the docker docs

**Comment by szaimen** [comment]
Hi @AriaanBruinsma, after reading your post, i get the impression that you were actually looking for this: https://github.com/nextcloud/all-in-one/tree/main/manual-install
